package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-native-svg-renderer",
   "version": "0.1.2",
-  "description": "Render an SVG Image from an URL",
+  "description": "Render a SVG Image from an URL",
```

Comment:
What about "Renders SVG Images from URLs"?

Reply:
Good idea!

Reply:
What do you think about "Renders SVG Images from URLs and plain SVG string"?